ts/components/LabelledItem.tsx

```diff
@@ -22,6 +23,14 @@ const styles = StyleSheet.create({
   },
   bottomLine: {
     borderBottomWidth: 1
+  },
+  inputStyle: {
+    height: 50,
+    paddingLeft: 5,
+    paddingRight: 5,
+    fontSize: 17,
+    color: "#5C6F82",
```

Comment:
either use a color from the theme `variables` or move this color there and give it a name